dxr_server/src/lib.rs

```diff
@@ -93,7 +94,10 @@ pub async fn server(handlers: HandlerMap, body: &str, headers: HeaderMap) -> (St
 
     let handler = match handlers.get(call.name()) {
         Some(handler) => handler,
-        None => return fault_to_response(404, "Unknown method."),
+        None => {
+            log::warn!("Missing rpc handler for {}", call.name());
```

Comment:
Same applies here.

It might also be good to put the static message prefix into a `const INVALID_HANDLER_MSG: &str = "..."; ` at the top of the file that can referenced by both log statements, so that it doesn't need to be kept in sync in two places.